R/shinywrappers.R

```diff
@@ -583,15 +578,8 @@ createRenderPrintPromiseDomain <- function(width) {
 renderText <- function(expr, env=parent.frame(), quoted=FALSE,
                        outputArgs=list(), sep=" ") {
 
-  if (!missing(env) || !missing(quoted)) {
-    deprecatedEnvQuotedMessage()
-    if (!quoted) expr <- enexpr(expr)
-    q <- new_quosure(expr, env)
-  } else {
-    q <- enquo(expr)
-  }
-
-  func <- quoToFunction(q, "renderText")
+  expr <- get_quosure(expr, env, quoted)
+  func <- quoToFunction(expr, "renderText")
```

Comment:
This cadence is repeated a bunch. Is this easy to wrap up into a helper method?

Reply:
(`expr` is never used afterwards either)

Reply:
I wanted to make it clear that what you get out of `get_quosure()` is a quosure. The `quoToFunction` function is relatively straightforward, but `get_quosure()` is kind of magical, and I wanted to keep the magical part contained.

Reply:
Thanks!